app/models/analysis.js

```diff
@@ -117,11 +126,12 @@ AnalysisSchema.statics.usageStatistics = function(cb) {
           }
         )
         .exec(function(err, items) {
+          client.set(self.collection.name + "_job_stats", JSON.stringify(items), (err, reply) => {});
```

Comment:
It might be best to use `this.cachePath()` instead of `self.collection.name+"_job_stats"`